src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectHandler.cs

```diff
@@ -139,108 +147,133 @@ public OpenIdConnectHandler(HttpClient backchannel, HtmlEncoder htmlEncoder)
         }
 
         /// <summary>
-        /// Handles Signout
+        /// Sign out the Relying Party from the OpenID provider 
```

Comment:
nit: this comment feels a bit misleading, as this method redirects the user agent TO the identity provider. It's not involved when the signout is triggered by the identity provider itself.